src/components/GlobalNavbar/GlobalNavbar.tsx

```diff
@@ -6,8 +6,7 @@ import useIsAuthenticated from 'contexts/auth/useIsAuthenticated';
 function GlobalNavbar() {
   const isAuthenticated = useIsAuthenticated();
 
-  // hiding navbar for now
-  return true ? null : (
+  return (
```

Comment:
in master, GlobalNavbar is rendered in AppContainer, which was removed from main routes: https://github.com/gallery-so/gallery/blob/main/src/scenes/PossiblyDeprecated_AppContainer/PossiblyDeprecated_AppContainer.tsx



should be able to render the GlobalNavbar somewhere directly in [Routes](https://github.com/gallery-so/gallery/blob/main/src/scenes/Routes.tsx), just above the <Router> but within <Location>

example: https://codesandbox.io/s/w2m1m67nj5?file=/src/App.js